src/streamlink_cli/utils/player.py

```diff
@@ -36,3 +38,20 @@ def find_default_player():
     if path:
         # Quote command because it can contain space
         return subprocess.list2cmdline([path])
+
+def sanitizeTitle(wholeTitle):
+    if wholeTitle is None:
+        wholeTitle = DEFAULT_FORMAT_ARGUMENTS["title"]
+    
+    if not is_win32:
+        if is_py2:
+            title = shlex_quote(wholeTitle.encode('utf8'))
+        elif is_py3:
+            title = shlex_quote(wholeTitle)
+    else:
+        if is_py2:
+            title = subprocess.list2cmdline([wholeTitle.encode('utf8')])
+        elif is_py3:
+            title = subprocess.list2cmdline([wholeTitle])
+    title = title.replace("$","$$")
```

Comment:
Don't know which replacement you're referring to. I factored sanitizeTitle() out of _create_arguments() because we use it twice in _create_arguments(). We use it twice in _create_arguments so that the user isn't forced to use @@ wrapping.